typegate/tests/runtimes/python_wasi/python_wasi_test.ts

```diff
@@ -241,3 +241,57 @@ Meta.test("Python WASI: infinite loop or similar", async (t) => {
   //     .on(e);
   // });
 }, { sanitizeOps: false });
+
+Meta.test("Python WASI: typegate reloading", async (metaTest) => {
+  const load = async () => {
+    const e = await metaTest.engine("runtimes/python_wasi/python_wasi.ts");
+    return e;
+  };
+  const engine = await load();
+  await metaTest.should("work before typegate is reloaded", async () => {
+    await gql`
+      query {
+        identityDef(
+          input: {
+            a: "hello",
+            b: [1, 2, "three"]
+          }) {
+          a
+          b
+        }
```

Comment:
can you please also add a test for python modules?
`def` and `lambda` functions are both registered on the default vm.

Reply:
on it.

Reply:
Done.